### ISSUE TITLE
Rover Motors Sporadically Fail to Operate

### ISSUE THREAD

**Issue by bwwagner** [bwwagner/SP17-barbot#8]
**Description:**  
The rover wheels will occasionally fail to operate during normal loop execution. The wheels will sporadically work several times, then fail to work for a long period of time.  Resetting the system through disconnecting the power and hitting the switch fails to resume operation of the wheels. The wheels will unpredictably restart once code is re-uploaded to the Aurduino board.

**Possible Causes:**
 - Hardware failure to communicate due to blocking voltage
 - Incorrect Startup Sequence in software
 - Temperature variance causing pins to expand/contract
 - Code written to bad sector of memory
 - Small break in a wire

**Comment by bwwagner** [comment]
https://github.com/adafruit

**Comment by bwwagner** [comment]
*Solution:*
This issue was solved by re-soldering a pin on the lower left side of the board.
 
We tracked this issue down to an unreliable connection between the Adafruit shield and the Arduino mainboard. The motors were observed to stop operation when the Adafruit shield was shifted by hand. Upon further inspection, it was discovered that one of the pins was soldered at a different length than the others.

While testing to solve this issue we confirmed continuity of the wires above the board and the breadboard.
Additionally, we discovered that using the other type of shield board caused a conflict when operating the rover and arm simultaneously. The motors would buzz loudly and operate very slowly from only the arm's begin() method.
